.github/workflows/check_install.yml

```diff
@@ -0,0 +1,40 @@
+name: Install and Import
+
+on:
+  push:
+    branches:
+    - main
+    - v0.*.x
+    tags:
+    - v*
+  pull_request:
+  workflow_dispatch:
+
+permissions: {}  # disables all GitHub permissions for the workflow
+
+jobs:
+  pip-install-and-import:
+    name: Installation with pip & import (py-${{ matrix.python }})
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.8', '3.10', '3.11', '3.12', '3.13']
```

Comment:
🌠 If we wanted to be ambitious and thorough later on (not in this PR!), the `lowest-direct` resolution strategy of `uv` would let us test `plasmapy_sphinx` against the oldest allowed versions of direct dependencies. 🤔

Reply:
Future work... I just want to get this to the next step first.